models/pod.go

```diff
@@ -81,15 +83,19 @@ func (pod *Pod) Parse(p *core_v1.Pod) {
 		if err == nil {
 			for _, name := range scs.InitContainers {
 				container := ContainerInfo{
-					Name:  name,
-					Image: lookupImage(name, p.Spec.InitContainers)}
+					Name:    name,
+					Image:   lookupImage(name, p.Spec.InitContainers),
+					IsProxy: false,
```

Comment:
Note that if you propose the "IsProxy" attribute/field to mark "this is container is a Proxy", the Istio init-container is typically a proxy as well: 
```
    image: docker.io/istio/proxyv2:1.9.1
    imagePullPolicy: Always
    name: istio-init
```

Reply:
Good point, I will makethe change.  The logs generated by the init-container are unlikely to benefit from access-log parsing but it is a proxy and should be flagged correctly for client-side consumption.  I will make the change.